### PR TITLE
Create initial README.md

### PR DESCRIPTION
**Pull request by anthonylavado**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h1 align="center">Jellyfin OpenSubtitles Plugin</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.media">Jellyfin Project</a></h3>
```
Comment (dkanada) [inline README.md]:
```suggestion
<h3 align="center">Part of the <a href="https://jellyfin.org">Jellyfin Project</a></h3>
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<p align="center">
+This is a plugin built with .NET that can download subtitles from OpenSubtitles.org for your videos.
```
Comment (dkanada) [inline README.md]:
```suggestion
This is a plugin built with .NET that can download subtitles from https://opensubtitles.org for your videos.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
```
Comment (dkanada) [inline README.md]: don't technically need a newline here

File: README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+</p>
+
+## Build Process
+1. Clone or download this repository
+2. Ensure you have .NET Core SDK setup and installed
+3. Build plugin with following command.
+```sh
+dotnet publish --configuration Release --output bin
+```
+4. Place the resulting .dll file in a folder called ```plugins/``` under  the program data directory or inside the portable install directory
```
Comment (dkanada) [inline README.md]:
```suggestion
4. Place the resulting .dll file in a folder called `plugins/` under the program data directory or inside the portable install directory
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+<p align="center">
+This is a plugin built with .NET that can download subtitles from OpenSubtitles.org for your videos.
+
+</p>
+
+## Build Process
+1. Clone or download this repository
+2. Ensure you have .NET Core SDK setup and installed
+3. Build plugin with following command.
```
Comment (dkanada) [inline README.md]:
```suggestion
3. Build plugin with following command
```

File: README.md (hunk 3 above)
Comment (anthonylavado) [reply README.md]: This was a copypasta fail

**Pull request merged**